index.js

```diff
@@ -7,6 +7,8 @@ require('dotenv').config();
 const app = express();
 const port = process.env.PORT || 8080;
 
+global.__basedir = __dirname + "/..";
```

Comment:
Would you happen to know where we are using this?

Reply:
Not sure...

Reply:
I think we can delete this. I believe I wrote this previously for the middleware